RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -1942,16 +1953,19 @@ class DeepTauId : public deep_tau::DeepTauBase {
           sp.scale(ele.deltaPhiSuperClusterTrackAtVtx(), dnn::ele_deltaPhiSuperClusterTrackAtVtx - e_index_offset);
       get(dnn::ele_deltaPhiSeedClusterTrackAtCalo + fill_index_offset_e) =
           sp.scale(ele.deltaPhiSeedClusterTrackAtCalo(), dnn::ele_deltaPhiSeedClusterTrackAtCalo - e_index_offset);
-      get(dnn::ele_mvaInput_earlyBrem + fill_index_offset_e) =
+      const bool mva_valid = (ele.mvaInput().earlyBrem > -2) || (year_ != 2026); // Known issue that input can be invalid in Phase2 samples (early/lateBrem==-2, hadEnergy==0, sigmaEtaEta/deltaEta==3.40282e+38). Unknown if also in Run2/3, so don't change there
+      if (mva_valid) {
```

Comment:
Which values are set if `mva_valid == false`? Is it checked that some safe defaults are used rather than values from previous electron candidate?

Reply:
OK, thanks. It makes sense.